### PR TITLE
Added django3.1 amd 3.2 support. Added tox and enable github actions …

### PR DESCRIPTION
**Pull request by awais786**
- Added django3.1 amd 3.2 testing.
- Added tox and enable github actions for different django and python versions.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -33,9 +39,12 @@
         'Framework :: Django :: 2.1',
         'Framework :: Django :: 2.2',
         'Framework :: Django :: 3.0',
+        'Framework :: Django :: 3.1',
+        'Framework :: Django :: 3.2',
     ],
     install_requires=[
         "django-compressor>=1.3",
         "libsass>=0.7.0,<1",
+        "django-appconf==1.0.3",
```
Comment (awais786) [inline setup.py:48]: https://github.com/django-compressor/django-appconf/pull/60
Known issue.

**Comment by awais786** [comment]
@gasman Please review this.

File: setup.py [hunk 2]
```diff
@@ -33,9 +39,12 @@
         'Framework :: Django :: 2.1',
         'Framework :: Django :: 2.2',
         'Framework :: Django :: 3.0',
+        'Framework :: Django :: 3.1',
+        'Framework :: Django :: 3.2',
     ],
     install_requires=[
         "django-compressor>=1.3",
         "libsass>=0.7.0,<1",
+        "django-appconf==1.0.3 ; python_version<'3'",
```
Comment (awais786) [inline setup.py:48]: https://github.com/django-compressor/django-appconf/pull/60
Known issue.

**Comment by gasman** [comment]
Thanks @awais786!

**Pull request merged**